packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -3957,6 +3959,44 @@ describe('StripeHelper', () => {
       });
     }
 
+    for (const type of [
+      'payment_method.attached',
+      'payment_method.automatically_updated',
+      'payment_method.updated',
+    ]) {
```

Comment:
I didn't see tests for the Firestore processing for `payment_method.*` webhooks in general, so I added some.